db/migrate/20170718111834_unify_container_definition_and_container.rb

```diff
@@ -0,0 +1,174 @@
+class UnifyContainerDefinitionAndContainer < ActiveRecord::Migration[5.0]
+  class ContainerDefinition < ActiveRecord::Base
+    has_many :container_port_configs, :class_name => 'UnifyContainerDefinitionAndContainer::ContainerPortConfig'
+    has_many :container_env_vars, :class_name => 'UnifyContainerDefinitionAndContainer::ContainerEnvVar'
+    has_one :security_context, :as => :resource
+    has_one :container, :class_name => 'UnifyContainerDefinitionAndContainer::Container'
+  end
+
+  class Container < ActiveRecord::Base
+    belongs_to :container_definition, :class_name => 'UnifyContainerDefinitionAndContainer::ContainerDefinition'
+  end
+
+  class SecurityContext < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    belongs_to :resource, :polymorphic => true
```

Comment:
Same comment as https://github.com/ManageIQ/manageiq-schema/pull/24/files#r129430898 (this is the other half)